src/vulnpy/flask/blueprint.py

```diff
@@ -34,6 +38,42 @@ def _cmdi_subprocess_popen():
     return get_template("cmdi.html")
 
 
+@vulnerable_blueprint.route(
```

Comment:
sorry, now that I looked at the frontend I noticed something else too - can you provide a `/deserialization/` endpoint just like `/cmdi/`? It would just render the `deserialization.html` template and nothing else. Once we can combine the unit tests to be framework-agnostic I think we won't run into cases like this as easily